diff --git a/pyobjc-core/setup.py b/pyobjc-core/setup.py
index 6183ba97e..e20a6db05 100644
--- a/pyobjc-core/setup.py
+++ b/pyobjc-core/setup.py
@@ -81,9 +81,9 @@ CFLAGS = [
     # "-fsanitize=address", "-fsanitize=undefined", "-fno-sanitize=vptr",
     # "--analyze",
     "-Werror",
-    "-I/usr/include/ffi",
     # "-fvisibility=hidden",
     # "-O3", "-flto",
+    "-I"+os.path.expandvars('$PREFIX')+"/include",
 ]
 
 # CFLAGS for other (test) extensions:
@@ -98,6 +98,7 @@ OBJC_LDFLAGS = [
     "-framework",
     "Carbon",
     # "-fvisibility=protected",
+    "-L"+os.path.expandvars('$PREFIX')+"/lib",
     "-g",
     "-lffi",
     # "-fsanitize=address", "-fsanitize=undefined", "-fno-sanitize=vptr",
@@ -387,6 +388,8 @@ def _find_executable(executable):
 
 
 def _working_compiler(executable):
+    print(f"executable: {executable}")
+
     if executable == "xcrun":
         return True
 
@@ -406,6 +409,8 @@ def _working_compiler(executable):
         stdout, stderr = p.communicate()
         status = p.wait()
         if status != 0:
+            print(f"stdout:\n{stdout}")
+            print(f"stderr:\n{stderr}")
             return False
 
         binfile = fp.name[:-1] + "o"
