diff --git a/pyobjc-core/setup.py b/pyobjc-core/setup.py
index 7d62ca29f..ee777beac 100644
--- a/pyobjc-core/setup.py
+++ b/pyobjc-core/setup.py
@@ -81,10 +81,11 @@ CFLAGS = [
     # "-fsanitize=address", "-fsanitize=undefined", "-fno-sanitize=vptr",
     # "--analyze",
     "-Werror",
-    "-I/usr/include/ffi",
     # "-fvisibility=hidden",
     "-O3",
     "-flto",
+    "-I${PREFIX}/include",
+    "-L${PREFIX}/lib",
 ]
 
 # CFLAGS for other (test) extensions:
