diff --git a/pyobjc-core/setup.py b/pyobjc-core/setup.py
index 6183ba97e..a0498e12f 100644
--- a/pyobjc-core/setup.py
+++ b/pyobjc-core/setup.py
@@ -81,9 +81,10 @@ CFLAGS = [
     # "-fsanitize=address", "-fsanitize=undefined", "-fno-sanitize=vptr",
     # "--analyze",
     "-Werror",
-    "-I/usr/include/ffi",
     # "-fvisibility=hidden",
     # "-O3", "-flto",
+    "-I"+os.path.expandvars('$CONDA_PREFIX')+"/include",
+    "-L"+os.path.expandvars('$CONDA_PREFIX')+"/lib",
 ]
 
 # CFLAGS for other (test) extensions:
